### ISSUE TITLE
Shelf is prone to bowing and shearing

### ISSUE THREAD

**Issue by NameOfTheDragon** [ALittleSlow/Ultimate-Drawer-System#2]
The geometry of the shelf renders it prone to bowing and sheering under any stress, which means the drawers can "let go" and fall out of the runners. Having printed two shelves, one 1U and one 1.5U, both 12U tall, and clipped them together, the whole stack is very easily pushed sideways into a parallellogram, which increases the space between the sides and can cause the drawers to fall out of their runners. Any load placed on the top tends to make the walls bow out with the same effect.

There is a lot of space at the back and some braces in there, or even a completely solid back panel, could make a big difference.

**Comment by ALittleSlow** [comment]
I agree. I have always printed the larger sizes with the back locking mechanism, even though I don't use the lock. This provides the necessary stiffness. However, my max height has been 6U.

**Comment by NameOfTheDragon** [comment]
> I agree. I have always printed the larger sizes with the back locking mechanism, even though I don't use the lock. This provides the necessary stiffness. However, my max height has been 6U.

I initially did a 4U shelf as a test print and that came out fine, so yes it seems to not scale very well. I think I'm going to have a go at a completely new design.
